tests/test_resource_requests.py

```diff
@@ -255,11 +255,6 @@ def test_with_column_folded_rayrunner_class(enable_actor_pool):
         "more_memory_request",
         assert_resources_1(col("id"), num_cpus=1, memory=5_000_000),
     )
-    assert_resources_2 = assert_resources.override_options(num_cpus=1, memory_bytes=None)
-    df = df.with_column(
-        "more_cpu_request",
-        assert_resources_2(col("id"), num_cpus=1),
-    )
```

Comment:
Why rm?

Reply:
See my note above
> Note:
> - One of the tests was cut down because it needed 3 CPUs to run, which combined with the 1 CPU for the scheduler actor, meant no more tasks could even be scheduled. I plan on adding some more informative errors/warnings when this happens in a future PR